### PR TITLE
feat: pizza login success page style refresh

### LINKED ISSUE #91: Feature: give `pizza login` page a refresh
### Suggested solution

This is the current page that gets displayed after a successful login:

Screenshot 2024-08-26 at 1 35 01 PM

This was built very early on without much thought for design: we should give this page a refresh. It's just basic HTML and CSS with a small `script` for the countdown: https://github.com/open-sauced/pizza-cli/blob/beta/cmd/auth/success.html

In my opinion, the simpler the better: it probably doesn't even need a countdown timer.

cc @isabensusan if you have any design ideas

### PR DESCRIPTION
**Pull request by nickytonline**
## Description

Gives a style refresh on the login success page.

TODO, add the nice pizza piece background from the OpenSauced homepage.

## Related Tickets & Documents

Closes #91

## Mobile & Desktop Screenshots/Recordings

**Before**

CleanShot 2024-08-29 at 10 44 24

**After**

CleanShot 2024-08-29 at 11 10 44

## Steps to QA

1. Open the web page api/auth/success.html in a browser

## Tier (staff will fill in)

- [ ] Tier 1
- [ ] Tier 2
- [ ] Tier 3
- [x] Tier 4

### CODE REVIEW

File: api/auth/success.html [hunk 1]
```diff
@@ -1,123 +1,38 @@
...
+
+    .container {
+      display: grid;
+      place-content: center;
+      height: calc(100vh - 5.5rem);
+      overflow: hidden;
+      font-size: 1.5rem;
+    }
+  </style>
+  <meta http-equiv="refresh" content="10;url=https://opensauced.pizza/" />
```
Comment (jpmcb) [inline api/auth/success.html:22]: Should this redirect them to the `app.opensauced.pizza`? Landing page may not bring that much value for newly logged in users?

**Review by jpmcb — APPROVED** [review]
Looks great!

File: api/auth/success.html (hunk 1 above)
Comment (nickytonline) [reply api/auth/success.html:22]: oops, yep.

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**